### PR TITLE
Add basic acceptance Test

### PR DESCRIPTION
**Pull request by Valantin**
#### Pull Request (PR) description
Add basic acceptance to check package and service

### CODE REVIEW

File: spec/acceptance/supportedos_spec.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+describe 'openvmtools class' do
+  context 'default parameters' do
+    it 'work with no errors' do
+      pp = <<-EOS
+        class { 'openvmtools': }
+      EOS
+
+      apply_manifest(pp, catch_failures: true)
+    end
```
Comment (ekohl) [inline spec/acceptance/supportedos_spec.rb:13]: We have https://github.com/voxpupuli/voxpupuli-acceptance/?tab=readme-ov-file#an-idempotent-resource and you can also write that shorter:
```suggestion
    it_behaves_like 'an idempotent resource' do
      let(:manifest) { 'include openvmtools' }
    end
```

**Review by ekohl — COMMENTED** [review]
Thanks for doing this!

File: spec/acceptance/supportedos_spec.rb (hunk 1 above)
Comment (Valantin) [reply spec/acceptance/supportedos_spec.rb:13]: eh, I know but it not run indempotent, because the service not start causing a systemd unit check

Comment (ekohl) [reply spec/acceptance/supportedos_spec.rb:13]: Could you then leave that as a comment in the file why it's not being checked for idempotency?

Comment (Valantin) [reply spec/acceptance/supportedos_spec.rb]: Done

**Review by ekohl — APPROVED** [review]

**Pull request merged**